lib/node/rpc.js

```diff
@@ -2643,9 +2633,7 @@ class RPC extends RPCBase {
 
     return {
       txid: tx.txid(),
-      hash: tx.wtxid(),
```

Comment:
If some API is using `.hash`, it might break. Have you checked no one's using this? If not, set this to `txid`

Reply:
It is used by RPC